tcms/rpc/tests/test_attachment.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=attribute-defined-outside-init, invalid-name, objects-update-used
+
+from tcms.rpc.tests.utils import APITestCase
+from tcms.tests.factories import TestPlanFactory
+
+class TestRemoveAttachment(APITestCase):
+    """Test for Attachment.remove_attachment"""
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.plan = TestPlanFactory()
+
+    def test_remove_attachment(self):
+        file_name = 'attachment.txt'
+
+        self.rpc_client.TestPlan.add_attachment(self.plan.pk, file_name, 'a2l3aXRjbXM=')
+        attachments = self.rpc_client.TestPlan.list_attachments(self.plan.pk)
```

Comment:
In this scenario that would happen during setup and then just try to remove the file and verify it's not there anymore.